Update/some_011.txt

```diff
@@ -1797,19 +1797,19 @@ void main()
 //r藤堂くんと一つ角の前で別れて、私は自宅に戻った。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "藤堂くんと一つ角の前で別れて、私は自宅に戻った。",
-			NULL, "Around the corner, Toudou-kun stopped before reaching my house.", Line_Normal);
+			NULL, "I parted ways with Toudou-kun in front of a corner, and returned home.", Line_Normal);
 	ClearMessage();
 
 //r最後まで家まで送る、と藤堂くんは言ってくれたけど、私のこんななりを見て、しかもクラスの男の子におぶられて帰ってきたとわかったら、おかーさんはきっといい顔をしないだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "最後まで家まで送る、と藤堂くんは言ってくれたけど、私のこんななりを見て、しかもクラスの男の子におぶられて帰ってきたとわかったら、おかーさんはきっといい顔をしないだろう。",
-			NULL, "Toudou-kun said that he would carry me all the way home, but I was sure that Mom would be unhappy if she saw this and realized I'd been spending time with a boy from school.", Line_Normal);
+			NULL, "Toudou-kun said that he would carry me all the way home, but I was sure that Mom would be unhappy if she saw me like this and realized I'd been spending time with a boy from school.", Line_Normal);
 	ClearMessage();
 
 //r…あるいは、藤堂くんに対してねぎらいよりも、理不尽な叱責をぶつけるかもしれない。だから、なんとか説得して帰ってもらった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…あるいは、藤堂くんに対してねぎらいよりも、理不尽な叱責をぶつけるかもしれない。だから、なんとか説得して帰ってもらった。",
-			NULL, "...Or maybe she would unreasonably reprimand Toudou-kun. So, somehow I persuaded him to let me go.", Line_Normal);
+			NULL, "...Or maybe she would reprimand Toudou-kun instead of thank him. So I managed to persuade him to let me go.", Line_Normal);
```

Comment:
```suggestion
			NULL, "...She might even reprimand Toudou-kun instead of thanking him. So I managed to persuade him to let me go.", Line_Normal);
```